capstone-backend/src/main/java/com/google/sps/data/Assignment.java

```diff
@@ -9,16 +9,18 @@
   private final String clubID;
   private final String whenCreated;
   private String whenDue;
+  private List<String> completedIDs;
 
-  public Assignment(String text, String clubID, String whenCreated, String whenDue) {
+  public Assignment(String text, String clubID, String whenCreated, String whenDue, List<String> completedIDs) {
     this.text = text;
     this.clubID = clubID;
     this.whenCreated = whenCreated;
     this.whenDue = whenDue;
+    this.completedIDs = completedIDs;
```

Comment:
Nice! Having each assignment hold the user IDs that have completed it make for a really clean solution!

Reply:
Thanks!